pymc/sampling_jax.py

```diff
@@ -136,9 +138,9 @@ def _get_log_likelihood(model: Model, samples, backend=None) -> Dict:
     """Compute log-likelihood for all observations"""
     data = {}
     for v in model.observed_RVs:
-        v_elemwise_logp = model.logp(v, sum=False)
-        jax_fn = get_jaxified_graph(inputs=model.value_vars, outputs=v_elemwise_logp)
-        result = jax.jit(jax.vmap(jax.vmap(jax_fn)), backend=backend)(*samples)[0]
+        v_elemwise_logpt = model.logpt(v, sum=False)
+        jax_fn = get_jaxified_graph(inputs=model.value_vars, outputs=v_elemwise_logpt)
+        result = jax.vmap(jax.vmap(jax_fn))(*jax.device_put(samples, jax.devices(backend)[0]))[0]
```

Comment:
loglikelihood should be fine with JIT, as non-elemwise logp has to be jittable for the model to sample in the first place

Reply:
it does not matter in here imo

Reply:
It should still be faster with jit even if we call it only once according to folks on the slack